src/utils/utils.ts

```diff
@@ -85,8 +101,15 @@ export const sanitizeTrendlineChildren = (
 };
 
 const isChartChildElement = (child: ChildElement<ChartChildElement> | undefined): child is ChartChildElement => {
-	return isRscComponent(child);
+	const res = isRscComponent(child);
+	console.warn('Is the child an rsc component?', res);
+	return res;
 };
+
+const isBigNumberChildElement = (child: ChildElement<BigNumberChildElement> | undefined): child is BigNumberChildElement => {
```

Comment:
We can replace the usage of this function with `isChartChildElement` since their implementation is the same.